### PR TITLE
DOC-2213 added backup descr for prod checklist

### PR DESCRIPTION
**Pull request by ashleykwtam**
Address DOC-2213, DOC-688
- Added description and links for production purposes

Q's
- Where should the backup section go on the page? Is there a specific order the headings are following for the existing content on the page?
- How detailed should the section be? The page already looks to have a lot of information and the current sections vary in lengths

Staging:
- [Production Settings](https://deploy-preview-15707--cockroachdb-docs.netlify.app/docs/stable/recommended-production-settings.html#backup-and-restore)

### CODE REVIEW

**Comment by kathancox** [comment]
Hey @ashleykwtam, these are good questions that we can sort out once we understand engineering/product's views on what we need to include here. I think this is a good start and touches on a few key points. Perhaps add Liv and then ask in the DR channel for an engineer's perspective here too? 

I'll complete a docs review once any prod/eng feedback is incorporated. Thanks so much for taking this one!

File: v22.2/recommended-production-settings.md [hunk 1]
```diff
@@ -314,6 +314,14 @@ For guidance on sizing, validating, and using connection pools with CockroachDB,
 
 {% include {{ page.version.version }}/prod-deployment/monitor-cluster.md %}
 
+## Backup and restore
+
+CockroachDB is built to be fault-tolerant and to recover automatically, but sometimes disasters happen. Having a [disaster recovery](disaster-recovery.html) plan enables you to recover quickly, while limiting the consequences.
```
Comment (livlobo) [inline v22.2/recommended-production-settings.md:319]: minor: "purpose-built"

**Review by livlobo — APPROVED** [review]
LGTM, thanks Ashley!

File: v22.1/recommended-production-settings.md [hunk 2]
```diff
@@ -312,6 +312,14 @@ For guidance on sizing, validating, and using connection pools with CockroachDB,
 
 {% include {{ page.version.version }}/prod-deployment/monitor-cluster.md %}
 
+## Backup and restore
+
+CockroachDB is built to be fault-tolerant and to recover automatically, but sometimes disasters happen. Having a [disaster recovery](disaster-recovery.html) plan enables you to recover quickly, while limiting the consequences.
+
+Taking regular backups of your data in production is an operational best practice. You can create [full](take-full-and-incremental-backups.html#full-backups) or [incremental](take-full-and-incremental-backups.html#incremental-backups) backups of a cluster, database, or table. We recommend taking backups to [cloud storage](use-cloud-storage-for-bulk-operations.html) and enabling object locking to protect the validity of your backups. CockroachDB supports Amazon S3, Azure Storage, and Google Cloud Storage for backups.
```
Comment (adityamaru) [inline v22.1/recommended-production-settings.md:319]: nit: should we link to https://www.cockroachlabs.com/docs/stable/use-cloud-storage-for-bulk-operations.html#object-locking when talking about object locking?

**Review by adityamaru — APPROVED** [review]
Nice! LGTM

**Comment by ashleykwtam** [comment]
@kathancox good for your review now, whenever you get a chance :)

**Review by kathancox — APPROVED** [review]
LGTM!

**Pull request merged**